src/ventures-for-shared-prosperity/index.njk

```diff
@@ -7,63 +7,60 @@ title: Ventures for Shared Prosperity
 {% block masthead %}
   {{ super() }}
   <div class="hero hero--centered standard-restricted-width">
+  <img src="" alt="colorful octogon"/>
+  <p>COHORT ONE: 2021-2022 </p>
     <h1 class="hero__title">
       Ventures for Shared Prosperity
     </h1>
+    <p> Improving the financial health and well-being of low and moderate income people in America.</p>
+    <div>Arrow here</div>
```

Comment:
Let's leave this `<div>Arrow here</div>` out for now because it's not text content. We'll likely accomplish this with CSS only.

Reply:
Fixed!